app.js

```diff
@@ -1,18 +1,35 @@
 const express = require('express');
 const { getTopics } = require('./controllers/topics.controller');
 const { getAPIDescription } = require('./controllers/endpoits.controller');
+const { getArticle } = require('./controllers/articles.controller');
 
 const app = express();
 
 app.get('/api/topics', getTopics);
 app.get('/api', getAPIDescription);
+app.get('/api/articles/:article_id', getArticle);
 
 app.all('*', (req, res) => {
   res.status(404).send({ msg: 'not found' });
 });
 
-app.use((err, request, response) => {
-  response.status(500).send({ msg: 'Internal server error!' });
+app.use((err, req, res, next) => {
+  if (err.status) {
```

Comment:
I would recommend checking for status and msg here as it is entirely possible that another kind of error could have a status code too, but no msg